### DISCUSSION TITLE
Can not checkout on Bitcraze store

### DISCUSSION THREAD

**Discussion by afwilkin** [bitcraze/discussions#1216] [General]
Unsure if anyone else is having a problem with checking out on the bitcraze store

It will only let me generate a quote

https://www.loom.com/share/e95a1e0fb9994b3887a228b57c731368

**Comment by afwilkin** [comment]
![image](https://github.com/bitcraze/discussions/assets/16963678/92388980-166f-4646-acdf-05a86e46225b)

**Comment by afwilkin** [comment]
I logged out, checked out as guest, and when i was on the store.checkout.io page I logged back in. worked out

I am the shopify whisperer

**Reply by knmcguire** [reply]
Nice! For anything store related, you should probably send an email next time to contact _at_ bitcraze.io :) Discussions is mostly meant for firmware/software/hardware related things.
